woocommerce/google-listings-and-ads
# Remove some `js/src/external-components`

### Changes proposed in this Pull Request:

This is a cleanup task, as the fixes for forked components were released.

- Update `@woocommerce/components` to the latest `6.2.0`  from `5.1.2` (7953722)
- Remove forked `ReportFilters` & co. (53ea575)
	as those fixes were released in `@woocommerce/components`:
	- woocommerce/woocommerce-admin#6890,
	- woocommerce/woocommerce-admin#6062
- Set static options in Programs autocomplete, (dda400e)
	as `Search` fix woocommerce/woocommerce-admin#6061 was released.

### Screenshots:




### Detailed test instructions:

According to the [woocommerce/components/CHANGELOG.md#620](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/CHANGELOG.md#620) the majority of changes were around `Search`, `SummaryNumber` and `<Filter>` components. So report pages are at the biggest risk, to be broken.

1. Go to report pages, or any other you may suspect :|
2. Check that it works as before.


### Changelog Note:

> Removed forked `@woocommerce/components`.

### Additional notes:
- Removing `external-components` makes it easier for us to catch up with other dependencies. For example, the latest `@woocommerce/eslint-plugin` complains a lot about external dependencies and experimental components being used. We have those problems also in our components, but this PR reduces a few.
